esignet-core/src/main/java/io/mosip/esignet/core/dto/ProxyAuthCodeRequest.java

```diff
@@ -0,0 +1,10 @@
+package io.mosip.esignet.core.dto;
+
+import lombok.Data;
+
+@Data
+public class ProxyAuthCodeRequest {
+
+    private String transactionId;
+    private String proxyAuthorizationCode;
```

Comment:
should we call it proxyAuthorizationCode or just authCode or code as part of oidc spec, as my understanding is that this API is generating a proxyAuthCode, not taking it as input.